src/pages/components/tooltip/style.mdx

```diff
@@ -11,13 +11,20 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 ### Standard tooltip
 
-| Element               | Property         | Color token           |
-| --------------------- | ---------------- | --------------------- |
-| Label                 | text color       | `$text-secondary`     |
-| Trigger button        | svg              | `$icon-secondary`     |
-| Trigger button: hover | svg              | `$icon-primary`       |
-| Container             | background-color | `$background-inverse` |
-| Text                  | color            | `$text-inverse`       |
+| Element        | Property         | Color token           |
+| -------------- | ---------------- | --------------------- |
+| Label          | text color       | `$text-secondary`     |
+| Trigger button | svg              | `$icon-secondary`     |
+| Container      | background-color | `$background-inverse` |
+| Text           | color            | `$text-inverse`       |
+
+#### Interactive states
```

Comment:
The "Interactive states" heading seems to have the incorrect padding between the heading and the table below it.